frontend/components/feedback-task/SearchBar.base.vue

```diff
@@ -92,15 +99,28 @@ export default {
     };
   },
   computed: {
+    isActiveSearchEmpty() {
+      return isNil(this.value) || this.value.length === 0;
+    },
     isSearchActive() {
-      return this.value?.length;
+      return !this.isActiveSearchEmpty;
+    },
+    searchValueIsEmpty() {
+      return isNil(this.searchValue) || this.searchValue.length === 0;
+    },
+    showDelete() {
+      if (isNil(this.searchValue)) return false;
+      if (this.isActiveSearchEmpty && this.searchValue.length === 0)
+        return false;
+
+      return true;
```

Comment:
is this equivalent?

```suggestion
      if (isNil(this.searchValue) || this.searchValue.length === 0) return false;
      return !this.isActiveSearchEmpty;
```

Reply:
!(this.isActiveSearchEmpty && this.searchValue`?`.length === 0); to add also the isNil test.
I make it in several lines instead of one to make it more readable. What is it better in this case ?